internal/policygen/policygen_test.go

```diff
@@ -49,90 +49,126 @@ func contains(s string, subs ...string) bool {
 
 // TestExamples is a basic test for the policygen to ensure it runs without error on the examples.
 func TestExamples(t *testing.T) {
-	exs, err := filepath.Glob("../../examples/policygen/*.yaml")
-	if err != nil {
-		t.Fatalf("filepath.Glob = %v", err)
-	}
 
-	if len(exs) == 0 {
-		t.Error("found no examples")
+	type wantDir struct {
+		path           string
+		wantFilesCount int
+		wantFiles      []string // Check some files.
 	}
 
-	for _, ex := range exs {
-		t.Run(filepath.Base(ex), func(t *testing.T) {
-			tmp, err := ioutil.TempDir("", "")
-			if err != nil {
-				t.Fatalf("ioutil.TempDir = %v", err)
-			}
-			defer os.RemoveAll(tmp)
-
-			args := &RunArgs{
-				ConfigPath: ex,
-				StatePath:  "../terraform/testdata/test.tfstate",
-				OutputPath: tmp,
-			}
-
-			if err := Run(&Fake{}, args); err != nil {
-				t.Fatalf("policygen.Run(%+v) = %v", args, err)
-			}
+	configPath := "../../examples/policygen/config.yaml"
+	common := []wantDir{
+		{
+			filepath.Join("forseti_policies", "overall"),
+			11,
+			[]string{
+				"bigquery_allow_locations.yaml",
+			}},
+		{
+			filepath.Join("gcp_org_policies"),
+			3,
+			[]string{
+				"main.tf",
+				"variables.tf",
+				"terraform.tfvars",
+			}},
+	}
 
-			// Check for the existence of policy folders and files.
-			dirs := []struct {
-				path           string
-				wantFilesCount int
-				wantFiles      []string // Check some files.
-			}{
-				{
-					filepath.Join(tmp, "forseti_policies", "overall"),
+	tests := []struct {
+		configPath string
+		statePath  string
+		wantDirs   []wantDir
+	}{
+		{
+			configPath,
```

Comment:
Same with `configPath`. This doesn't need to be part of the struct, since it's the same for all test cases.

Reply:
Done